### ISSUE TITLE
Simplify the interface

### ISSUE THREAD

**Issue by frandavid100** [syncthing-gtk/syncthing-gtk#32]
Having a GTK client for Syncthing is great, but some users may find the interface too complex for a Gnome app.

I have made a couple mockups of a simplified interface. For the sake of simplicity I have removed some capabilities, but if you think some (or all) of them should be included, I can try to add them back to the mockups.

![Untitled](https://user-images.githubusercontent.com/694794/236897311-048a91c9-d7bc-41f8-b81a-871bdd62b267.png)
![Untitled2](https://user-images.githubusercontent.com/694794/236897346-2c88b857-c579-4c88-9c5c-bf80de71f71b.png)
![Untitled3](https://user-images.githubusercontent.com/694794/236897375-62345cfa-83e9-4933-a4d5-be74115ff394.png)

**Comment by frandavid100** [comment]
Maybe related to #8?

**Comment by benjamin051000** [comment]
LOVE how this looks. Absolutely love it. Wow

**Comment by frandavid100** [comment]
Hey, Benjamin, so glad you feel that way. It seems the issue hasn't had any other activity since I opened it, though 🙁

**Comment by Rudd-O** [comment]
Whoa this is amazing.  Can we get this created?

**Comment by frandavid100** [comment]
Hey there, Rudd. I'm happy you like the design so much.

Is there any chance we can get some feedback on this, @Salamandar?

**Comment by Salamandar** [comment]
This is a huge revamp of the app. To be honest i’m not sure i’d have the time to do this. Though, I’d love to read some merge requests :D If someone more experienced than I am in GTK would like to try something, I’ll find the time to review changes.

**Comment by Salamandar** [comment]
Also that would go with #8

**Comment by frandavid100** [comment]
Hi, thanks for the reply.

Some time ago I opened [a similar thread on Gnome Discourse](https://discourse.gnome.org/t/proposal-integrate-syncthing-into-gnome-settings/15387/15?u=davidprieto) , except that one was about integrating this interface in Gnome Settings; and there was some interest in creating an app from scratch.

Maybe there could be some potential for collaboration here. Would you mind giving it a read, maybe talk to the people who have sort-of-offered to work on this if they have the time?

**Comment by Rudd-O** [comment]
I don't want to open the Settings app to see that stuff.  This is supposed to be a companion app to show the status, it shouldn't run embedded in a modal views app.

**Comment by frandavid100** [comment]
I mean, I think the mockups work both as a Settings panel and as a standalone app. I'll take whatever I can get.

**Comment by Salamandar** [comment]
Actually, the plan i have in mind for syncthing-gtk would be to split it into 2 projects :
* a python API client library (there already are some up-to-date libraries)
* the rest of the GTK app, just using the library

That way the maintenance would be way smaller. Also people could work on either project according to their competences.

**Comment by frandavid100** [comment]
That sounds great, and seems like it would make collaboration much easier. Let's say someone else wants to start working on the GTK frontend next week, what would need to happen before that?

**Comment by Salamandar** [comment]
Sorry for the delay. To be honest, that would involve a full revamp of the source code. So you could start to work on it today :) 
If you have experience with working with GTK4, you could start from scratch. I'm not sure the current source base is future-proof.
On the other side, there's the code that communicates with Syncthing. You shouldn't worry too much about that, I'd love to work with aiosyncthing (see #48).

**Comment by antonok-edm** [comment]
@frandavid100 see also https://github.com/syncthing-gtk/syncthing-gtk/issues/8#issuecomment-1612398263
